hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+| 149			| Module Specific 12			| ---------------------------------------------	|
+| 150			| Reserved						| ---------------------------------------------	|
+| 151			| GND							| GND											|
+| 152			| LCD B3						| PJ15 \| LCD B3								|
+| 153			| LCD B4						| PK3 \| LCD B4									|
+| 154			| LCD B5						| PK4 \| LCD B5									|
+| 155			| LCD B6						| PK5 \| LCD B6									|
+| 156			| LCD B7						| PK6 \| LCD B7									|
+| 157			| ADC F							| PC0 \| ADC10									|
+| 158			| ADC G							| PC2 \| ADC12									|
+| 159			| ADC H							| PC3 \| ADC13									|
+| 160			| 3.3V							| 3.3V											|
+| 161			| LCD G2						| PJ9 \| LCD G2									|
+| 162			| LCD G3						| PJ10 \| LCD G3								|
+| 163			| LCD G4						| PJ11 \| LCD G4								|
+| 164			| LCD G5						| PK0 \| LCD G5									|
+| 165			| LCD G6						| PK1 \| LCD G6									|
+| 166			| Module Specific 13			| ---------------------------------------------	|
+| 167			| Reserved						| ---------------------------------------------	|
+| 168			| LCD R7						| PJ6 \| LCD R7									|
+| 169			| GND							| GND											|
+| 170			| LCD G7						| PK2 \| LCD G7									|
+| 171			| LCD R3						| PJ2 \| LCD R3									|
+| 172			| LCD R4						| PJ3 \| LCD R4									|
+| 173			| LCD R5						| PJ4 \| LCD R5									|
+| 174			| LCD R6						| PJ5 \| LCD R6									|
+| 175			| SPI A SCK						| PB3 \| SPI1 SCK								|
+| 176			| SPI A MISO					| PB4 \| SPI1 MISO								|
+| 177			| Module Specific 14			| ---------------------------------------------	|
+| 178			| SPI A MOSI					| PB5 \| SPI1 MOSI								|
+| 179			| Module Specific 15			| ---------------------------------------------	|
+| 180			| 3.3V							| 3.3V											|
+| 181			| Module Specific 16			| ---------------------------------------------	|
+| 182			| Module Specific 17			| ---------------------------------------------	|
+| 183			| VBAT							| VBAT											|
+| 184			| Module Specific 18			| ---------------------------------------------	|
+| 185			| GND							| GND											|
+| 186			| GND							| GND											|
+| 187			| RESET							| RESET											|
+| 188			| USB Host D+					| PB15 \| USBH D+								|
+| 189			| JTAG RTCK						| ---------------------------------------------	|
+| 190			| USB Host D-					| PB14 \| USBH D-								|
+| 191			| JTAG TDO						| ---------------------------------------------	|
+| 192			| 3.3V							| 3.3V											|
+| 193			| JTAG NTRST					| ---------------------------------------------	|
+| 194			| USB Device D+					| PA12 \| USB D+								|
+| 195			| JTAG TDI						| ---------------------------------------------	|
+| 196			| USB Device D-					| PA11 \| USB D-								|
+| 197			| JTAG TCK (SWCLK)				| PA14 \| JTCK \| SWCLK							|
+| 198			| GND							| GND											|
+| 199			| JTAG TMS (SWDIO)				| PA13 \| JTMS \| SWDIO							|
+| 200			| Reserved						| ---------------------------------------------	|
+
+# Related Products
```

Comment:
Delete this whole section and put it on the intro page.

Reply:
I have the Standards on the intro. This is for the individual module. I think it would be too much to have both the standard and the individual module information on the same page.

Reply:
I didn't see anything talking about the dev or breakout on the intro page

Reply:
I thought you were referring to the table for the standard, not the related products part. I agree, this should be on the intro page.